src/main/java/com/mabl/integration/jenkins/MablStepBuilder.java

```diff
@@ -132,6 +134,14 @@ public void setAppBaseUrl(String appBaseUrl) {
         this.appBaseUrl = appBaseUrl;
     }
 
+    @DataBoundSetter
+    public void setWebUrlOverride(String webUrlOverride) {this.webUrlOverride = webUrlOverride;} // To change the webUrl dynamically if the user wants to change during deployment execution
+
+    @DataBoundSetter
+    public void setApiUrlOverride(String apiUrlOverride){
```

Comment:
Minor: can you move your inline code comment to be a Javadoc comment for the method iteself?